kdcproxy/config/__init__.py

```diff
@@ -49,14 +49,14 @@ def use_dns(self):
 
 class KDCProxyConfig(IConfig):
     GLOBAL = "global"
-    default_filename = "/etc/kdcproxy.conf"
+    default_filenames = ["/usr/local/etc/kdcproxy.conf", "/etc/kdcproxy.conf"]
 
     def __init__(self, filename=None):
         self.__cp = configparser.ConfigParser()
         if filename is None:
             filename = os.environ.get("KDCPROXY_CONFIG", None)
         if filename is None:
-            filename = self.default_filename
+            filename = self.default_filenames
```

Comment:
Please change this to `filenames` (and elsewhere)

Reply:
Will pick this up on Monday as soon as I get into the office.